auth-server/src/main/resources/application.yml

```
@@ -34,14 +31,47 @@ spring:
+    show-sql: true
     properties:
       hibernate:
-        dialect: org.hibernate.dialect.PostgreSQLDialect
+        format_sql: true
+  config:
+    import: "optional:configserver:http://configserver:8060/"
 
-    show-sql: true
+
+
+#Acuator#Acuator 
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)